packages/front-end/pages/experiments/index.tsx

```diff
@@ -725,6 +729,14 @@ const ExperimentsPage = (): React.ReactElement => {
                         onPageChange={setCurrentPage}
                       />
                     )}
+                    {canAddTemplate && !templates.length ? (
```

Comment:
I think we also want to limit this to only show once they have at least 5 experiments.  Before that point, templates are not really that valuable and this will distract from the main thing we want them to do - create their first few experiments.

Reply:
Sounds great - `pngquant` was able to reduce the size to 44K and I've updated the display logic.